msticpy/common/azure_auth_core.py

```diff
@@ -320,22 +333,6 @@ def _filter_all_warnings(record) -> bool:
     return True
 
 
-def _create_auth_options(cloud: str = None, tenant_id: str = None) -> dict:
```

Comment:
In the mp_config_load branch I've made some changes to this section to fix a bug. 
You can overwrite this section completely when you merge.